packages/compiler/src/server/serverlib.ts

```diff
@@ -860,14 +861,34 @@ export function createServer(host: ServerHost): Server {
       const tspDiag = currentDiagnosticIndex.get(vsDiag.data?.id);
       if (tspDiag === undefined || tspDiag.codefixes === undefined) continue;
 
+      if (
+        vsDiag.codeDescription?.href &&
+        tspDiag.codefixes?.findIndex((x) => x.id === "show-linter-rule-doc-url") === -1
+      ) {
+        mutate(tspDiag.codefixes).push(
+          defineCodeFix({
+            id: "show-linter-rule-doc-url",
```

Comment:
please extract the common string

Reply:
As comment above this still shouldn't even be a code fix